rust/main/agents/relayer/src/msg/op_submitter.rs

```diff
@@ -339,6 +352,20 @@ async fn prepare_task(
     // Prepare at most `max_batch_size` ops at a time to avoid getting rate-limited
     let ops_to_prepare = max_batch_size as usize;
     loop {
+        // Apply backpressure to the prepare queue if the submit queue is too long.
+        if let Some(max_len) = max_submit_queue_len() {
```

Comment:
should this default to 2-3x the max batch size?

Reply:
As idk if we'll use this going forward I think just will keep it manual for now, we can add later if it's useful though